profiles/kentik_snmp/apc/apc_ats.yml

```diff
@@ -0,0 +1,346 @@
+      enum:
+        atsPowerSupplyFailure: 1
+        atsPowerSupplyOK: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.10
+      name: atsStatusSwitchStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.11
+      name: atsStatusFrontPanel
+      enum:
+        locked: 1
+        unlocked: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.12
+      name: atsStatusSourceAStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.13
+      name: atsStatusSourceBStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.14
+      name: atsStatusPhaseSyncStatus
+      enum:
+        inSync: 1
+        outOfSync: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.15
+      name: atsStatusVoltageOutStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.16
+      name: atsStatusHardwareStatus
+      enum:
+        fail: 1
+        ok: 2
+
+  - MIB: PowerNet-MIB_ATS
+    table:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.3.2
+      name: atsInputTable
+    symbols:
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.2
+        name: atsNumInputPhases
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.3
+        name: atsInputVoltageOrientation
+        enum:
+          unknown: 1
+          singlePhase: 2
+          splitPhase: 3
+          threePhaseToNeutral: 4
+          threePhaseToPhase: 5
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.4
+        name: atsInputFrequency
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.5
+        name: atsInputType
+        enum:
+          unknown: 1
+          main: 2
+          bypass: 3
+    metric_tags:
+      - tag: input_index
+        column:
+          OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.1
+          name: atsInputTableIndex
+      - tag: input_name
+        column:
+          OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.6
+          name: atsInputName
+  - MIB: PowerNet-MIB_ATS
+    table:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.3.3
+      name: atsInputPhaseTable
+    symbols:
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.3
+        name: atsInputVoltage
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.4
+        name: atsInputMaxVoltage
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.5
+        name: atsInputMinVoltage
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.6
+        name: atsInputCurrent
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.7
+        name: atsInputMaxCurrent
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.8
+        name: atsInputMinCurrent
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.9
+        name: atsInputPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.10
+        name: atsInputMaxPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.3.1.11
+        name: atsInputMinPower
```

Comment:
most of these should all be metric tag values since they rarely change. i would leave these 3 as the primary metrics:

```
atsInputVoltage
atsInputCurrent
atsInputPower
```

Reply:
Updated in latest commit.